lib/restforce/db/client.rb

```diff
@@ -52,24 +52,22 @@ def initialize(**_)
       #     Time.now,
       #   )
       #
-      #   #=> #<Restforce::Mash
-      #          latestDateCovered="2015-05-18T22:31:00.000+0000"
-      #          earliestDateAvailable="2015-04-11T06:44:00.000+0000"
-      #          deletedRecords=[
-      #            #<Restforce::Mash
-      #              deletedDate="2015-05-18T22:31:17.000+0000"
-      #              id="a001a000001a5vOAAQ"
-      #            >
-      #          ]
-      #        >
+      #   #=> [
+      #         #<Restforce::Mash
+      #           deletedDate="2015-05-18T22:31:17.000+0000"
+      #           id="a001a000001a5vOAAQ"
+      #          >,
+      #       ]
       #
       # Returns a Restforce::Mash with a `deletedRecords` key.
```

Comment:
You're right; I caught the example above but neglected to change the documented return value.